### ISSUE TITLE
Cocoapod support

### ISSUE THREAD

**Issue by lampapos** [nearit/react-native-connectivity-status#10]
We use Cocapod as a package manager in our RN project (in the RN 0.60+ Cocoapods is used in all projects by default). `react-native-connectivity-status` doesn't contain `*.podspec` file so it can't be managed by Cocoapods and it creates some difficulties during project build.

Sorry, I've read your contribution guidelines after PR creation :) so I create this issue after PR (https://github.com/nearit/react-native-connectivity-status/pull/9).

### Environment

```
  React Native Environment Info:
    System:
      OS: macOS 10.14.6
      CPU: (12) x64 Intel(R) Core(TM) i7-8850H CPU @ 2.60GHz
      Memory: 66.28 MB / 16.00 GB
      Shell: 5.3 - /bin/zsh
    Binaries:
      Node: 12.6.0 - /usr/local/bin/node
      Yarn: 1.17.3 - /usr/local/bin/yarn
      npm: 6.9.0 - /usr/local/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    SDKs:
      iOS SDK:
        Platforms: iOS 12.4, macOS 10.14, tvOS 12.4, watchOS 5.3
      Android SDK:
        API Levels: 23, 26, 28
        Build Tools: 23.0.1, 26.0.1, 27.0.3, 28.0.3
        System Images: android-28 | Google APIs Intel x86 Atom
    IDEs:
      Xcode: 10.3/10G8 - /usr/bin/xcodebuild
    npmPackages:
      react: 16.8.3 => 16.8.3
      react-native: 0.59.10 => 0.59.10
    npmGlobalPackages:
      create-react-native-module: 0.5.0
      react-native-cli: 2.0.1
      react-native-create-library: 3.1.2
```

Target Platforms:
    iOS: 10.0

### Steps to Reproduce

1. `yarn add react-native-connectivity-status`
2. `react-native link react-native-connectivity-status`
3. `cd ios`
4. `pod install`

### Expected Behavior

If a project contains `Podfile` `react-native link` should add a line to it.

### Actual Behavior

Because the lib doesn't contain `*.podspec` `react-native link` directly changes `*.xcproject` file. This creates issues for Cocoapod (RN app just doesn't detect the native module on iOS).

**Comment by 14Lucky14** [comment]
I don't know why I'm getting these emails, I have no idea what they mean.
Please take me off your list. Thanks

**Comment by lampapos** [comment]
@14Lucky14 you are receiving these emails because you "is watching" a repository on GitHub - https://github.com/nearit/react-native-connectivity-status Go here and choose "Not watching" in the top panel

**Comment by 14Lucky14** [comment]
Thank you!

**Comment by federicoboschini** [comment]
Addressed in #9
